impeller/renderer/backend/vulkan/gpu_tracer_vk.cc

```diff
@@ -5,127 +5,153 @@
 #include "impeller/renderer/backend/vulkan/gpu_tracer_vk.h"
 
 #include <utility>
+#include "fml/logging.h"
 #include "fml/trace_event.h"
 #include "impeller/base/validation.h"
-#include "impeller/renderer/backend/vulkan/command_buffer_vk.h"
-#include "impeller/renderer/backend/vulkan/command_encoder_vk.h"
 #include "impeller/renderer/backend/vulkan/context_vk.h"
-#include "impeller/renderer/command_buffer.h"
+#include "vulkan/vulkan_enums.hpp"
```

Comment:
Is this right? Should this just be `vulkan/vulkan.hpp`?

Reply:
I get "included header is not used"